### DISCUSSION TITLE
Access Swell OpenAPI specification file

### DISCUSSION THREAD

**Discussion by mat250** [swellstores/community#74] [Help] [answered]
Hi Swell dev team, 

Mathieu here from the [Meta API](https://www.meta-api.io/) team.

Do you have any OpenAPI, Swagger or Postman collection for your API?
That would help us a lot to integration your API to [our catalog](https://dashboard.meta-api.io/connectors) and also help all developers using any API tools.

Cheers

**Comment by maarten2424** [comment, marked as answer]
Hey @mat250,
We don't have an OpenAPI / Swagger Spec for our API (yet). You can find our API endpoints trough our documentation: https://developers.swell.is/backend-api/introduction

Thanks!

**Comment by papertokyo** [comment]
@mat250 Would [this postman collection](https://github.com/swellstores/resource-kit/blob/main/postman-collection.json) be sufficient?

**Reply by mat250** [reply]
@papertokyo That's perfect! Thanks a lot.

I've been able to convert it and generate an OpenAPI file : https://gist.github.com/mat250/7e32a882ca75b56cd46d8f39665ac701

**Reply by greph** [reply]
Amazing! @mat250 , do you mind if we add this file to our internal resource kit repo (or share it with customers)?

**Reply by mat250** [reply]
You can share it :)
